googler

```diff
@@ -334,6 +334,15 @@ except GetoptError as e:
     print("googler:", e)
     sys.exit(1)
 
+# Set up logging
```

Comment:
> introducing a full framework for logging is a overkill.

Yeah probably. However,

> I don't want to execute a single line of code if it's not necessary.

as I said, `if debug` is basically implicit in `logger.debug`.

> Because it adds to maintenance too.

The basic logging interface was stable across Python 2 and 3 (we're not using any advanced feature here), and from all the cues I picked up over the years, there won't be as dramatic a breaking release as Python 3, ever. However, it is a valid argument that using a logging framework adds complexity that is probably not necessary.

Feel free to close this PR.